src/Illuminate/Database/Query/Builder.php

```diff
@@ -1505,6 +1506,185 @@ public function orWhereNotNull($column)
         return $this->whereNotNull($column, 'or');
     }
 
+    /**
+     * Add a where clause to determine if "date" column is in the past to the query.
+     *
+     * @param  string|array  $columns
+     * @param  \DateTimeInterface|string|null  $now
+     * @param  string  $boolean
+     * @param  bool  $not
+     * @return $this
+     */
+    public function wherePast($columns, $now = null, $boolean = 'and', $not = false)
+    {
+        $type = 'Basic';
+        $operator = $not ? '>=' : '<';
+        $value = $now ?? Carbon::now();
+
+        if ($value instanceof DateTimeInterface) {
+            $value = $value->format('Y-m-d H:i:s.u');
+        }
+
+        foreach (Arr::wrap($columns) as $column) {
+            $this->wheres[] = compact('type', 'column', 'boolean', 'operator', 'value');
+            $this->addBinding($value);
+        }
+
+        return $this;
+    }
```

Comment:
Why not just a shorthand for `whereDate()`?
```suggestion
    public function whereDateBefore($columns, $value = null, $boolean = 'and', $not = false)
    {
        return $this->whereDate($columns, $not ? '>=' : '<', $value, $boolean);
    }
```

Reply:
I could refactor to use `whereDate` if this didn't account the time. But it does.

Reply:
ah, thanks for the clarification 👍🏻